### DISCUSSION TITLE
How to make game ui like this?

### DISCUSSION THREAD

**Discussion by crapthings** [rexrainbow/phaser3-rex-notes#211] [Q&A]
![image](https://user-images.githubusercontent.com/1147704/146552402-3c8a6592-a8e1-4c89-acbd-3b173ea5f878.png)

![image](https://user-images.githubusercontent.com/1147704/146552492-7dba75fb-80ed-403a-aa2a-fbeb57326b8b.png)

![image](https://user-images.githubusercontent.com/1147704/146552813-cf0ab00e-da4e-49ea-8d0e-3c52c8dbd434.png)

how to position or layout UI like the above image

rexUI looks very promised, do we have a tutorial?

is this a label?

I mean if I have an icon and a custom text, should I use the label to achieve this

what about the layout and responsive size(scale)?

![image](https://user-images.githubusercontent.com/1147704/146553066-a7ab81f6-0be3-43a4-87c6-71f2f5b9294c.png)

**Comment by rexrainbow** [comment]
Label ui element has these parts 
- left side image
- center text
- right side image

It might not be enough in these cases. For example:
![image](https://user-images.githubusercontent.com/817962/146557245-5c4c111d-4126-49bd-94fd-1df794798b6c.png)
It has 
- left side image
- center 2 texts
- image pin at top-right corner

This ui element can be composed of 
- [OverlapSizer](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-overlapsizer/), which can align child element at any corner
    -  [Horizontal Sizer](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-sizer/), at center
        - Image
        -  [Vertical Sizer](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-sizer/), places
            - Text
            - Text
    - Image at right-top corner

**Reply by crapthings** [reply]
so overlap sizer is kind of flex, and horizontal sizer is a flex-row, and vertical sizer is flex-column

**Reply by rexrainbow** [reply]
Yes, they can map to these css styles
